openmc/bounding_box.py

```diff
@@ -95,9 +95,25 @@ def __or__(self, other: BoundingBox) -> BoundingBox:
         new |= other
         return new
 
-    def __contains__(self, point):
-        """Check whether or not a point is in the bounding box"""
-        return all(point > self.lower_left) and all(point < self.upper_right)
+    def __contains__(self, other):
+        """Check whether or not a point or another bounding box is in the bounding box.
+
+        For another bounding box to be in the parent it must lie fully inside of it,
+        with no overlap.
```

Comment:
I'm not sure if overlap is the right word here.

Reply:
Intersect?

Reply:
Nit-pick alert: I think the final phrase can simply be removed to make the statement less ambiguous.

Reply:
Yes, exactly